### ISSUE TITLE
Redesign "launch" blog post

### ISSUE THREAD

**Issue by mdlincoln** [programminghistorian/jekyll#497]
A few days after the soft launch of the redesign (tentatively scheduled for 2017-07-07) we would like to post a redesign announcement that highlights some of the biggest changes (in particular the enhanced lesson page). @fredgibbs & @amandavisconti co-authored?

**Comment by acrymble** [comment]
Is it possible for this to go out next week?

**Comment by fredgibbs** [comment]
that's the plan.

**Comment by mdlincoln** [comment]
@fredgibbs will post draft of the general blog post for the team to look over. One this general one goes live, I'll post #500 about a week later.

**Comment by fredgibbs** [comment]
i'm waiting on this a little longer since i think i can finish filtering/sorting state maintenance in the next few days, which will be nice to have in place before we attract attention to the post and the lessons page. stay tuned!

**Comment by fredgibbs** [comment]
OK, I've posted a draft in the _posts folder https://github.com/programminghistorian/jekyll/blob/gh-pages/_posts/2017-07-28-new-lessons-page.md; please anyone who's interested have a quick read and make any changes that seem useful. Don't be shy about editing! Maybe sometime this afternoon we can switch published to true if that gives everyone enough time for review.

**Comment by mdlincoln** [comment]
I'd love to add some screenshots (or better yet, some GIFs) showing the buttons in action. I can do that now.

**Comment by mdlincoln** [comment]
I think it reads great, @fredgibbs

**Comment by fredgibbs** [comment]
thanks @mdlincoln for the GIFs and screenshot! Huge improvement. I'm happy to flip the published flag whenever.

**Comment by mdlincoln** [comment]
😬 oops, I think I accidentally set it to published already when I was trying to preview it locally! So... I think this is now posted.

**Comment by fredgibbs** [comment]
haha--perfect! one less thing to do now.
